templates/types/streaming/nextjs/app/components/ui/chat/index.ts

```diff
@@ -28,17 +28,17 @@ export const DOCUMENT_FILE_TYPES: DocumentFileType[] = [
 ];
 
 export type UploadedFileMeta = {
-  id: string;
   name: string; // The uploaded file name in the backend (including uuid and sanitized)
```

Comment:
for what are we actually needing this? it's a bit confusing having a name and a filename